Makefile

```diff
@@ -168,10 +154,14 @@ $(KIND_BIN): $(TESTBIN_DIR)
 	@kubectl cluster-info
 
 clean: export KUBECONFIG = $(KIND_KUBECONFIG)
-clean:
+clean: ## Cleans up the generated resources
 	$(KIND_BIN) delete cluster --name $(KIND_CLUSTER) || true
 	docker stop "$(KIND_REGISTRY_NAME)" || true
 	docker rm "$(KIND_REGISTRY_NAME)" || true
 	docker rmi "$(E2E_IMG)" || true
 	rm -r testbin/ dist/ bin/ cover.out $(BIN_FILENAME) || true
 	$(MAKE) -C e2e clean
+
+.PHONY: help
+help: ## Show this help
+	@egrep -h '\s##\s' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
```

Comment:
`egrep` is actually deprecated. Can you modify the command to work with `grep -E` instead? see https://man7.org/linux/man-pages/man1/egrep.1.html